target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz7490.dts

```diff
@@ -0,0 +1,282 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7490";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_info_red;
+		led-running = &led_power;
+		led-upgrade = &led_info_red;
+
+		led-dsl = &led_internet;
+		led-internet = &led_internet;
```

Comment:
I'm not sure whether it's a good idea two have the same LED here, maybe just use led-dsl and remove led-internet.